examples/2.1.5.1.8-facet-json-query.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+require_once(__DIR__.'/init.php');
+htmlHeader();
+
+// create a client instance
+$client = new Solarium\Client($adapter, $eventDispatcher, $config);
+
+// get a select query instance
+$query = $client->createSelect();
+
+// get the facetset component
+$facetSet = $query->getFacetSet();
+
+// create a json query instance and set options
+$inStockFacet = new Solarium\Component\Facet\JsonQuery(['local_key' => 'stock_query', 'query' => 'inStock:true']);
+
+// add json query instance to the facetSet
+$facetSet->addFacet($inStockFacet);
+
+// this executes the query and returns the result
+$resultset = $client->select($query);
+
+// display the total number of documents found by solr
+echo 'NumFound: '.$resultset->getNumFound();
+
+// JsonQuery returns a FacetSet, not Buckets like a jsonTerms facet
+$stock_query = $resultset->getFacetSet()->getFacet('stock_query');
+
+// the facet count is returned as a 'count' sub-facet
```

Comment:
In search_api_solr we use this comment:
    
```
// The implicit "count" aggregation over all results matching the query
// exists only any JSONFacet set.
```

I think we should comment like this:
```
// A JSONFacet triggers an implicit "count" aggregation over all results which is accessible
// just like a Facet within the FacetSet.
```

Reply:
Ah, except I'm not sure 'all results' is quite the right phrase as this is the count for the JsonQuery alone, not all results?
Perhaps:
// A JsonQuery triggers an implicit "count" aggregation for the result of the query which is accessible just like a Facet within the FacetSet.

Reply:
sounds good :-)